Installers/Linux/RUN/postinstall.sh

```diff
@@ -16,8 +16,18 @@ fi
 DIR=$(pwd)
 IDIR="/usr/lib/mono/xbuild/MonoGame/v3.0"
 
-# Automatic dependency installer
-./Dependencies/dependencies.sh
+# Show dependency list
+echo "Please make sure the following packages are installed:"
+echo " - monodevelop"
```

Comment:
@cra0zy don't they need this before they install? Otherwise the .mpack will not get setup?
Any idea how to deal with that.